pubspec.yaml

```diff
@@ -30,15 +30,19 @@ environment:
 dependencies:
   flutter:
     sdk: flutter
-  cloud_firestore: ^4.8.2
-  firebase_storage: ^11.1.2
+  cloud_firestore: ^4.9.1
+  firebase_storage: ^11.2.6
   flutter_facebook_auth: ^6.0.0
```

Comment:
do we need it?

Reply:
yes, we need the firestore cloud for the database, and firebase storage to upload and store the image file to get a URL and that URL will be saved in the database.